cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModelStats.java

```diff
@@ -279,13 +282,23 @@ private void utilizationForResources(ClusterModel clusterModel, OptimizationOpti
           double capacity = resource.isHostResource() ? broker.host().capacityFor(resource)
                                                       : broker.capacityFor(resource);
           double utilizationPercentage = utilization / capacity;
+          if (utilizationPercentage > lowUtilizationThreshold) {
```

Comment:
What if the `lowUtilizationThreshold` and `utilizationPercentage` are both 0? Wouldn't `>=` be a sufficient condition?

Reply:
Right